config/locales/sv.yml

```diff
@@ -1683,6 +1683,13 @@ sv:
 
           must_be_logged_in: *must_be_logged_in_as_admin
 
+      members_need_packets:
+        subject: Members need packets sent
+        message_text:
+          message_intro: "These members need membership packets sent to them:"
+          membership start date: membership start
+        error_sending: "ERROR! There was a problem sending the 'Members need packets' alert email"
```

Comment:
FEL! Det blev ett fel när 'Members need packets' mailet skulle skickas 

( I dont know if this one needs translating?)